core/src/main/scala/org/apache/spark/scheduler/sparrow/SparrowSchedulerBackend.scala

```diff
@@ -129,18 +129,18 @@ class SparrowSchedulerBackend(scheduler: SparrowScheduler)
             context.senderAddress
           }
           logInfo(s"Registered executor $executorRef with ID $executorId")
-          addressToExecutorId(executorRef.address) = executorId
```

Comment:
i basically didnt take these changes, since i wanted to check why they are needed